scripts/tests.sh

```diff
@@ -124,21 +124,13 @@ function download_aleth()
     elif [ -z $CI ]; then
         ALETH_PATH="aleth"
     else
-        # Any time the hash is updated here, the "Running compiler tests" section should also be updated.
         mkdir -p /tmp/test
-        if grep -i trusty /etc/lsb-release >/dev/null 2>&1
-        then
-            # built from d661ac4fec0aeffbedcdc195f67f5ded0c798278 at 2018-06-20
-            ALETH_BINARY=aleth_2018-06-20_trusty
-            ALETH_HASH="54b8a5455e45b295e3a962f353ff8f1580ed106c"
-        else
-            # built from d661ac4fec0aeffbedcdc195f67f5ded0c798278 at 2018-06-20
-            ALETH_BINARY=aleth_2018-06-20_artful
-            ALETH_HASH="02e6c4b3d98299885e73f7db6c9e3fbe3d66d444"
-        fi
-        ALETH_PATH="/tmp/test/aleth"
-        wget -q -O $ALETH_PATH https://github.com/ethereum/cpp-ethereum/releases/download/solidityTester/$ALETH_BINARY
-        test "$(shasum $ALETH_PATH)" = "$ALETH_HASH  $ALETH_PATH"
+        # Any time the hash is updated here, the "Running compiler tests" section should also be updated.
+        ALETH_HASH="8db2019d3185c78e3043f211d0a360aff450a7f4"
+        wget -q -O /tmp/test/aleth.tar.gz https://github.com/ethereum/aleth/releases/download/v1.4.0rc7/aleth-1.4.0rc6+commit.d7e94caa-linux-x86_64.tar.gz
```

Comment:
There was a "better" release of rc7 which actually had the version bump internally :)

Reply:
Ok - but more importantly (at least if rc7 hasn't changed otherwise): there are still random test failures that I can't explain. I think I'll need to have another look when I'm back (unless someone wants to take over while I'm away).

Reply:
Surprisingly the random failures happen *only* on circle, not on travis so far.

Reply:
Oh also `1.4.x` doesn't support Constantinople, while we have added those opcodes, so we need to use `1.5.x`.

Reply:
PR uses 1.5.x now.